pages/HomePage.js

```diff
@@ -13,6 +14,8 @@ export default function HomePage() {
         <hr />
         <Events />
         <hr />
+        <Resources />
```

Comment:
we should also include links to the following resources: how to prep for interviews, how to contact professors, where to apply to internships, and we should also link some of the powerpoints that we have in our drive that could be helpful.